executor.go

```diff
@@ -194,6 +196,12 @@ func (e *executor) executeCall(ctx context.Context, index string, c *pql.Call, s
 	case "TopN":
 		e.Holder.Stats.CountWithCustomTags(c.Name, 1, 1.0, []string{indexTag})
 		return e.executeTopN(ctx, index, c, shards, opt)
+	case "Rows":
```

Comment:
Copied comments from previous PR:

	
  @travisturner
travisturner 27 days ago  Member
What do you think about this being called Column instead of Rows? In that case it would be similar to the Row query. Additionally, instead of returning RowIDs it could return a *Column (or Column) potentially containing IDs and attributes. I haven't thought this through very much, so I'm really just throwing out for discussion.

  @tgruben
tgruben 26 days ago  Member
I like the concept of Column returning all the details about a specific column (rows and attributes). I think the label breaks down when we are trying to interact with all/some rows contained in a field. I feel we need an additional term for dealing with the field based issues. Perhaps introduce some ...Info based terms ie FieldInfo, ColumnInfo,RowInfo. Still feels hackish, hoping this will trigger some synapse action elsewhere.